libdevcrypto/SecretStore.cpp

```diff
@@ -84,35 +86,37 @@ static js::mValue upgraded(std::string const& _s)
 	return js::mValue();
 }
 
-SecretStore::SecretStore(std::string const& _path): m_path(_path)
+SecretStore::SecretStore(string const& _path): m_path(_path)
 {
-	load();
+	if (!m_path.empty())
```

Comment:
It is not strictly necessary. If no path is provided (which is not the case in real use of this class), it just does not store the keys permanently.